accelerator/apis.c

```diff
@@ -25,6 +25,20 @@ int apis_logger_release() {
   return 0;
 }
 
+status_t api_get_ta_info(char** json_result) {
+  status_t ret = SC_OK;
+
+  ret = ta_generate_info_serialize(json_result);
+  if (ret != SC_OK) {
+    ret = SC_TA_OOM;
+    log_error(apis_logger_id, "[%s:%d:%s]\n", __func__, __LINE__, "SC_TA_OOM");
+    goto done;
+  }
+
+done:
```

Comment:
The goto label is not needed. Modify to:
```
...
  if (ret != SC_OK) {
    ret = SC_TA_OOM;
    log_error(apis_logger_id, "[%s:%d:%s]\n", __func__, __LINE__, "SC_TA_OOM");
  }

  return ret;
```

Reply:
Copy that!